ai/core/src/unitycatalog/ai/test_utils/client_utils.py

```diff
@@ -25,9 +25,19 @@ def client() -> DatabricksFunctionClient:
     if TEST_IN_DATABRICKS:
         return DatabricksFunctionClient(profile=PROFILE)
     else:
-        with mock.patch(
-            "unitycatalog.ai.core.databricks.get_default_databricks_workspace_client",
-            return_value=mock.Mock(),
+        with (
+            mock.patch(
+                "unitycatalog.ai.core.databricks.get_default_databricks_workspace_client",
+                return_value=mock.Mock(),
+            ),
+            mock.patch(
+                "unitycatalog.ai.core.databricks._validate_databricks_connect_available",
```

Comment:
Do we need this mock?

Reply:
haha no. completely internal to any access that we're making. thanks for the catch :)